README.md

```diff
@@ -12,17 +12,20 @@ using [custom resource definitions (CRDs)](https://kubernetes.io/docs/tasks/acce
 ## Getting Started
 
 The KubeSlice Controller orchestrates the creation and management of slices on worker clusters.
-It is strongly recommended to use a released version. Follow the instructions provided in this [document](https://docs.avesha.io/documentation/open-source/0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller).
+It is strongly recommended to use a released version. Follow the instructions provided in this [document](https://kubeslice.io/documentation/open-source/0.5.0/cloud-cluster-quick-start).
 
-## Building and Deploying `kubeslice-controller` in a Local Kind Cluster
-For more information, see [getting started with kind clusters](https://docs.avesha.io/documentation/open-source/0.2.0/getting-started-with-kind-clusters).
+## Building and Deploying a KubeSlice Controller on a Local Kind Cluster
+
+See our documentation on [installing the KubeSlice Controller on a Kind Cluster](https://kubeslice.io/documentation/open-source/0.5.0/tutorials/kind-install-kubeslice-controller).
```

Comment:
on a kind cluster

Reply:
done

Reply:
on a kind cluster - not `on a Kind Cluster`

Reply:
updated